sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -15,11 +15,11 @@ export interface CreateSpanFunctionArgs {
   /**
    * Package name prefix
    */
-  packagePrefix: string;
+  packagePrefix: string | undefined;
```

Comment:
Can we add a ref comment about when to use `undefined`?

Reply:
Comment added (although now it's empty string as the sentintel instead)